import-service/importProductsFile/handler.js

```diff
@@ -0,0 +1,55 @@
+const AWS = require("aws-sdk");
+const BUCKET = "rsschool-node-aws-s3-task5";
+const log = require("../logger");
+const util = require("util");
+const merge = require("lodash/merge");
+const corsResponse = require("../corsResponse");
+
+module.exports = {
+  importProductsFile: async function (event) {
+    log.info(`importProductsFile function is called with args: ${util.inspect(event)}`);
+    const s3 = new AWS.S3({ region: "us-east-1" });
+
+    const queryStringParams = event.queryStringParameters || {};
+    const fileName = queryStringParams.name; //catalog.csv
+    if (!fileName) {
+      log.info(`Query parameter 'name' not found`);
+      return merge(corsResponse, {
+        statusCode: 400,
+        body: JSON.stringify({
+          ok: false,
+          message: "Query parameter 'name' not found",
+        }),
+      });
+    }
+
+    const params = {
+      Bucket: BUCKET,
+      Key: "uploaded/" + fileName,
+      Expires: 60,
+      ContentType: "text/csv",
+    };
+
+    let signedUrl = "";
```

Comment:
Why not const? If something wrong with `signedUrl` U shouldn't pass it to user

Reply:
Thanks, refactored